bin/fusor-undercloud-configurator

```diff
@@ -8,6 +8,48 @@ def getHwAddr(ifname):
   info = fcntl.ioctl(s.fileno(), 0x8927,  struct.pack('256s', ifname[:15]))
   return ':'.join(['%02x' % ord(char) for char in info[18:24]])
 
+def isInvalidIP(ip):
+  if re.match('^(?:[0-9]{1,3}\.){3}[0-9]{1,3}$', ip):
+    return False
+  return True
```

Comment:
Why isn't this 
```return not re.match('^(?:[0-9]{1,3}\.){3}[0-9]{1,3}$', ip)```

If it matches return false, if it doesn't match return true. By negating the match you turn this into a one line function.